custom-commands/hospitality/deployment/DEPLOYREADME.md

```diff
@@ -1,32 +1,76 @@
+# Deploying a Sample Custom Command Application to Your Azure Subscription
+
+<p align="center">UNDER DEVELOPMENT</p>
+
 ## Things you will need
-* An Azure subscription. [Azure](https://portal.azure.com)
-* Powershell 6.0 or greater [Download Powershell](https://github.com/PowerShell/PowerShell/releases)
-* Azure CLI [Download Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). This was made using version 2.4.0
-* .Net Core SDK [Install SDK](https://docs.microsoft.com/en-us/dotnet/core/install/sdk?pivots=os-windows)
+* An Azure account. [sign up for free](https://azure.microsoft.com/free/ai/).
+* Powershell 6.0 or greater. [Download Powershell here](https://github.com/PowerShell/PowerShell/releases). 
+    * On Windows, download and run an .msi file (e.g. "PowerShell-7.0.0-win-x64.msi")
+* Azure CLI. [Insall Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) version 2.4.0 or higher.
+* .NET Core SDK 2.1 or higher. [Install SDK here](https://docs.microsoft.com/en-us/dotnet/core/install/sdk?pivots=os-windows).
+
+## What are you deploying?
+ 
+ TODO: Need diagram of CC and Visualization, and a description of the components and how they interact
 
 ## Deploying Azure Resources
-Clone the repository and change directory so you are in the root/custom-commands/hospitality/deployment folder.
+Clone the repository and change directory so you are in the \<<repo-root\>>\custom-commands\hospitality\deployment folder.
 
-You will need to unrestrict powershell's script execution policy by running the following in an administrator powershell
+You will need to unrestrict powershell's script execution policy by running the following in an administrator powershell:
+```cmd
```

Comment:
this should be powershell not cmd

Reply:
Not sure there is a powershell tag for GitHub markdown, but I'll look.

Reply:
I just don't want to confuse people.

Reply:
```powershell 
This is available.

Reply:
Frustrating that there is no official documentation on this... This thread is relevant and seems to suggest ```powershell or ```ps. I switched to ```powershell
https://stackoverflow.com/questions/20303826/highlight-bash-shell-code-in-markdown